### PR TITLE
Update brakeman to 4.3.0

### PR DESCRIPTION
**Pull request by mrfyda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,17 @@ docker run -it -v $srcDir:/src  <DOCKER_NAME>:<DOCKER_VERSION>
 We use the [codacy-plugins-test](https://github.com/codacy/codacy-plugins-test) to test our external tools integration.
 You can follow the instructions there to make sure your tool is working as expected.
 
+## Generating documentation
+
+Update brakeman version in `docs/Gemfile` and then run:
```
Comment (rtfpessoa) [inline README.md]: Can you check the way we do it on rubocop? There we use the same file to setup the tool locally and inside the docker.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,19 @@ docker run -it -v $srcDir:/src  <DOCKER_NAME>:<DOCKER_VERSION>
 We use the [codacy-plugins-test](https://github.com/codacy/codacy-plugins-test) to test our external tools integration.
 You can follow the instructions there to make sure your tool is working as expected.
 
+## Update Version with Docs
+
+Change the version in `.brakeman-version` and then run on this project root:
+
+```sh
+bundle clean
```
Comment (rtfpessoa) [inline README.md]: Why do you need to set a custom install path inside the sources? Doesn't it work like https://github.com/codacy/codacy-rubocop#update-version-with-docs ?

File: build.sbt [hunk 3]
```diff
@@ -26,26 +29,45 @@ version in Docker := "1.0"
 
 organization := "com.codacy"
 
-val brakemanVersion = "3.3.0"
+lazy val toolVersion = TaskKey[String]("Retrieve the version of the underlying tool from patterns.json")
```
Comment (rtfpessoa) [inline build.sbt]: This is wrong, the correct syntax is `lazy val toolVersion = TaskKey[String]("toolVersion", "Retrieve the version of the underlying tool from patterns.json")`. I had some problems and this fixed it.

File: circle.yml [hunk 4]
```diff
@@ -1,26 +1,37 @@
 machine:
```
Comment (rtfpessoa) [inline circle.yml]: Since you are touching this file can you use https://github.com/codacy/codacy-cppcheck/blob/master/.circleci/config.yml ?

**Review by rtfpessoa — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mrfyda) [reply README.md]: The script scrapes the source files for additional information

Comment (rtfpessoa) [reply README.md]: Hmmm, then I would prefer to put it in `/tmp`. Putting stuff inside resources is not nice. Also if we do not need to ruby scripts to run we can put then in the root of the project.

Comment (mrfyda) [reply README.md]: "if we do not need to ruby scripts to run" i don't get it. what do you mean? they are not added to the docker image.

**Review by rtfpessoa — APPROVED** [review]
LGTM 👍

**Pull request merged**